test/app/src/testApp.js

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* global process, window, document */
+/* eslint-disable no-console */
+import OktaAuth from '@okta/okta-auth-js';
+
+/* eslint-disable prefer-destructuring */
+const DOMAIN = process.env.DOMAIN;
+const CLIENT_ID = process.env.CLIENT_ID;
+/* eslint-enable prefer-destructuring */
+
+const ISSUER = `https://${DOMAIN}/oauth2/default`;
+const HOST = window.location.host;
+const REDIRECT_URI = `http://${HOST}/implicit/callback`;
+
+const oktaAuth = new OktaAuth({
+  // maxClockSkew: Number.MAX_SAFE_INTEGER, // FORCES TOKEN RENEW
```

Comment:
I'm assuming this is left here deliberately as it's helpful, but I'm mentioning it in case I'm wrong.